source/math/FloatBox2D.ooc

```diff
@@ -111,6 +111,15 @@ FloatBox2D: cover {
 				result add(i)
 		result
 	}
+	distance: func (point: FloatPoint2D) -> FloatSize2D {
+		((point - this center) toFloatSize2D() absolute - this size / 2.f) maximum(FloatSize2D new())
```

Comment:
Do you mind adding a pair of parentheses here to make it more clear what you want to divide with `.2f`?